site/_includes/homepage/signup.html

```diff
@@ -7,6 +7,9 @@ <h2>Find out about threats to our basic rights.</h2>
   <div class="privacy">
     We will only email when we have something important or awesome to tell
     you. <a href="/privacy/">Privacy.</a>
+    <div class="push-top">
```

Comment:
Normally I would use a `p` tag here, but that comes with extra width and layout styles in this section. Instead I'm using a `div` to avoid a larger refactor.